docs/source/getting_started/project_structure.rst

```diff
@@ -33,6 +33,14 @@ The repository directory (or root directory) contains the following:
 
   - This file details the licences the repository uses.
 
+- ``LICENCE-CONTENT.md``
+
+  - This file details the Creative Commons Attribution-ShareAlike 4.0 International license.
```

Comment:
Suggestion: Specify that LICENCE-CONTENT is for the content in markdown files, while LICENCE is for the structure